### ISSUE TITLE
Priority 1 accessibility issue: Duplicate ID value

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6545]
**What**

The ID associated with the ‘Close notification’ button is duplicated. Having duplicated ID values can cause anomalous behaviour with assistive technology as they sometimes rely on IDs to utilise certain functionalities.

ID value(s):
id="icon-cross-title"

Solution:
Ensure that ID values are unique to avoid potential anomalous behaviour for users of assistive technology.

**Where**

URL: https://wellcomecollection.org/whats-on/today Page: What's on | Wellcome Collection
URL: https://wellcomecollection.org/access Page: Accessibility | Wellcome Collection 
This issue occurs throughout the website.

<img width="186" alt="Screenshot 2021-05-27 at 11 19 44" src="https://user-images.githubusercontent.com/7315126/119809888-7252a200-bedd-11eb-98e8-73cfeac59abf.png">

**Comment by gestchild** [comment]
unable to find a duplicate id on the page
